solr/core/src/test/org/apache/solr/response/transform/TestSubQueryTransformerDistrib.java

```diff
@@ -134,6 +135,7 @@ public static void setupCluster() throws Exception {
   }
 
   @SuppressWarnings("serial")
+  @SuppressForbidden(reason = "java.net.URL ctors deprecated since Java 20")
```

Comment:
You fix some and suppress others.  Is there a pattern/approach to it?  (just curious; maybe not)